deploy/deployment.yaml

```diff
@@ -17,22 +17,17 @@ spec:
       - name: rbac-query-proxy
```

Comment:
How do we use `deploy` folder? is it only for testing?

Reply:
yes